src/server.js

```diff
@@ -1,13 +1,49 @@
-const express = require('express');
-const dotenv = require('dotenv');
+import '@babel/polyfill';
+import morgan from 'morgan';
+import express from'express';
+import { json } from 'express';
+import dotenv from 'dotenv';
+import colors from 'colors';
+import logger from './App/middlewares/logger';
+import routes from './App/routes/index';
 
+
+//Load env vars
+dotenv.config();
+
+//Initialising the app with express
 const app = express();
 
-//Body parser
-app.use(express.json());
+//Initialise logger middleware
+app.use(logger);
+
+//
+app.use(json());
+
+// Mounting Morgan
+app.use(morgan('dev'));
 
+//Mounting routes
+app.use('/', routes);
```

Comment:
You can use this to remove repetitions in `src/routes/index.js` by moving the repeated  part here 
```suggestion
app.use('/api/v1/techblogs', routes);
```